src/V3Coverage.cpp

```diff
@@ -872,26 +873,28 @@ class CoverageVisitor final : public VNVisitor {
         lineTrack(nodep);
     }
 
-    // Lambdas not supported for expression coverage
-    void visit(AstWith* nodep) override {
-        VL_RESTORER(m_seeking);
-        if (m_seeking == SEEKING) abortExprCoverage();
-        m_seeking = ABORTED;
-        iterateChildren(nodep);
-        lineTrack(nodep);
+    void visit(AstFuncRef* nodep) override {
+        if (nodep->taskp()->lifetime().isAutomatic()) {
+            visit(static_cast<AstNodeExpr*>(nodep));
+        } else {
+            exprUnsupported(nodep);
+        }
     }
 
     void visit(AstNodeExpr* nodep) override {
         if (m_seeking != SEEKING) {
             iterateChildren(nodep);
         } else {
             std::stringstream emitV;
-            V3EmitV::verilogForTree(nodep, emitV);
-            // Add new expression with a single term
-            CoverExpr expr;
-            expr.emplace_back(nodep, m_objective, emitV.str());
-            m_exprs.push_back(std::move(expr));
-            checkMaxExprs();
+            if (V3EmitV::verilogForTree(nodep, emitV, true)) {
```

Comment:
I don't think this should couple to V3EmitV like this.  I'd prefer we let anything bad through and fix the issues.  We could have a message, e.g. from V3SImulate:

```
#ifdef VL_DEBUG
            static std::set<VNType> s_typePrinted;
            const auto pair = s_typePrinted.emplace(nodep->type());
            if (pair.second)
                UINFO(0,
                      "Unknown node type in SimulateVisitor: " << nodep->prettyTypeName() << endl);
#endif
```